const a = () => {
    throw new Error('this is the error happened in b.js');
}

a()